cli/actions/__tests__/build.test.js

```diff
@@ -0,0 +1,74 @@
+const shell = {
```

Comment:
should we move this somewhere more global? seems like a lot of the files would need this mock